vertica_backup/backup.py

```diff
@@ -154,10 +158,11 @@ def main(argv=None):
         delete_pickles(fs_store)
         delete_pickles(swift_store, config['retain'])
 
-    except:
+    except Exception as e:
         log.exception('Unhandled Exception in Backup upload')
         # Move the Epoch files back to their original names so a retry run does not encounter issues with them
-        epoch_files.restore()
+        if None != epoch_files:
```

Comment:
Minor: The idomatic way with python is a 'if epoch_files is not None"
https://www.python.org/dev/peps/pep-0008/#programming-recommendations